style.css

```diff
@@ -93,10 +79,36 @@ img:hover{
   float: left;
 }
 
-#closing{
-  color:rgb(231,84,128);
+#closing {
+  color: rgb(231, 84, 128);
 }
 
-#candidates{
+#candidates {
   border: 10px solid pink;
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 600px) {

```

Comment:
When using @media queries, you must ensure your rules are placed _after_ the default ones you want to override, because barring any differences in selector precedence, the last one will be applied.

If you wanted to take a mobile-first approach, you would specify your mobile styling first as the default and then define any uoles for tablet/desktop browsers using media queries